File: library/src/main/java/me/shkschneider/skeleton/java/ObjectHelper.java
package me.shkschneider.skeleton.java;

import android.support.annotation.NonNull;

import com.google.gson.Gson;

public class ObjectHelper {

    protected ObjectHelper() {
        // Empty
    }

    public static String jsonify(@NonNull final Object object) {
        return new Gson().toJson(object, object.getClass());
    }

}

File: library/src/main/java/shkschneider/skeleton/java/ObjectHelper.kt
package me.shkschneider.skeleton.java

import com.google.gson.Gson

object ObjectHelper {

    @Deprecated("Broken (with Koltin).", ReplaceWith("Gson.toJson()", "com.google.gson.Gson"))
    fun jsonify(obj: Any?): String {
        obj ?: return "{}"
        return Gson().toJson(obj, obj::class.java)
    }

}
